test/storage/keymanager.test.ts

```diff
@@ -34,15 +34,16 @@ describe('Keymanager class', () => {
     });
 
     test('generatePrivateKeyFromPassword() returns privatekey', async () => {
-        const password = '123';
+        const password = 'above day fever lemon piano sport';
         const { privateKey, salt } = await generatePrivateKeyFromPassword(password);
 
         expect(privateKey).toBeInstanceOf(PrivateKey);
+        console.log('privateKey', privateKey);
         expect(salt).toBeDefined();
     });
 
     test('time hashing in generatePrivateKeyFromPassword() function', async () => {
-        const password = '123';
+        const password = 'above day fever lemon piano sport';
         const salt = Checksum256.from(randomBytes(32));
         const options = {
             salt: Buffer.from(salt.hexString, 'hex'),
```

Comment:
it would be good to Make a new test "generatePrivateKeyFromPassword() function Takes > 2s" to generate the privately from the passphrase. It should use the argon settings we are going to use in production and time how long it takes, and check that that is greater than two seconds.

 When you run this, can you please tell me how long the function takes to execute?